robot_smach_states/test/human_interaction/human_interaction.py

```diff
@@ -2,11 +2,12 @@
 import unittest
 
 # TU/e Robotics
+from hmi import TimeoutException, HMIResult
 from robot_skills.mockbot import Mockbot
 
 # Robot smach states
 import robot_smach_states.util.designators as ds
-from robot_smach_states.human_interaction.human_interaction import Say
+from robot_smach_states.human_interaction.human_interaction import Say, HearOptions, WaitForPersonInFront, AskYesNo
 
 
 class TestSay(unittest.TestCase):
```

Comment:
This test currently fails

Reply:
Samesies as the last one: it succeeds for me, what is the error you get?

Reply:
CI succeeds

Reply:
FAIL: test_ask_yes_no (__main__.TestHearOptions)

Traceback (most recent call last):
  File "human_interaction.py", line 83, in test_ask_yes_no
    self.assertEqual(state.execute(), option)
AssertionError: 'no' != 'yes'


FAIL: test_say (__main__.TestSay)

Traceback (most recent call last):
  File "human_interaction.py", line 23, in test_say
    self.assertIn(sentence, self.robot.speech.speak.call_args_list[-1].args)
AssertionError: 'This is an easy sentence' not found in args


Ran 5 tests in 0.271s

FAILED (failures=2)

Reply:
This error seems to be a typo. since the solution is "Tech United" and we assert "tech united"

Reply:
`test_ask_yes_no` failed with me too. I fixed it.

`test_say ` didn't fail with me.

Reply:
How is it possible that the test failed but CI still succeeds? that seems like something that shouldn't happen.

Reply:
I think the tests in de subdirectories are skipped at the moment

Reply:
All sub dirs require a `__init__.py` and all the file need to state with `test`

Reply:
You're sure about the `__init__`? Doesn't really make sense to me.... Can imagine the `test_` prefix though...

Reply:
Yes, I tested it and the number of tests were lower when removing the `__init__.py` in a subfolder.

Reply:
In Python, an `__init__.py` file in a directory makes it a python package (that is, Python looks for additional python modules in such a package directory). In most cases, the `__init__.py` file can be empty (it is loaded when you import the package rather than a module in the package).

Reply:
> How is it possible that the test failed but CI still succeeds? that seems like something that shouldn't happen.

Likely the used set of software is not exactly the same. Truly equal setups at every computer system that one uses is a non-trivial problem.

A setup not only includes the software in the repository but also anything else that it uses directly or indirectly, such as other packages that you checked out (eg from a different branch), your Python environment, installed third-party software, or the operating system. Tests involving timing may even depend on the computer being used to test.